src/components/DateInput/TimePicker.tsx

```diff
@@ -49,7 +63,20 @@ const TimePicker: React.FC<TimePickerProps> = ({ date, onTimeUpdate, label = ''
 
   return (
     <>
+      {label && !is12Hours && (
+        <Box
+          as="label"
+          flexShrink={0}
+          fontWeight="bold"
+          role="status"
+          pr={4}
+          htmlFor={slugify(slugify(hoursLabel))}
```

Comment:
there's an un-needed double `slugify` invocation here :)
```suggestion
          htmlFor={slugify(hoursLabel)}
```